### DISCUSSION TITLE
Media Entities - Not updating status or attributes

### DISCUSSION THREAD

**Discussion by ian-tait** [alandtse/alexa_media_player#1544] [Q&A]
Hello All,

Media Player is not updating states/attributes as playlists progress.
The only time Media Player updates the states are when HA is rebooted, or the interaction is "reloaded".

I was on the assumption this was to update every couple of seconds as there is a section for the song progress bar.

Has anyone else come across this before, and did you fix it?
After reading the issues on GitHub for Media Player, I have tried to use the older & new login methods. 
Overall, the old method seemed to show better results, but the updates are still not working.

I would like to get this fixed.

Currently running:
Home Assistant 2022.3.5
Alexa Media Player v3.11.0

Thank you,
Ian

**Comment by J-T-W** [comment]
#1600 - Saw this after I posted.  Same basic issue.

**Reply by Jadefalcon100** [reply]
Same for me. 

Login Method is definitely working (new access token every hour in logs) but the media player entities stop updating after a while ...

**Reply by moritzbindewald** [reply]
I have the same issue
